### PR TITLE
Sign In Gate / Long mandatory sign in AB test dcr

### PR DESCRIPTION
**Pull request by vlbee**
## What does this change?

Test definitions and set up for long-running mandatory sign in gate trial

Depends on Frontend PR: https://github.com/guardian/frontend/pull/25514 (AB Switches)

_Note:_ the mandatory variant will only show if all cmp consents are accepted.

Updated test audience percentages[ here](https://docs.google.com/spreadsheets/d/1bfxdnPOmqRKCIUaZLxYyHVAmVrmXlNpsUPkAdk0d7S8/edit#gid=970675296&range=I36)

4x Control tests overlap
4x Variant tests overlap

### Tested

- [x] On CODE

## Screenshots

**Variant (non-dismissable mandatory gate**) 
image

**Control (current gate)**
image

### CODE REVIEW

File: dotcom-rendering/src/web/experiments/tests/sign-in-gate-mandatory-long-test.ts [hunk 1]
```diff
@@ -0,0 +1,244 @@
+import type { ABTest } from '@guardian/ab-core';
+import { getCountryCodeSync } from '../../lib/getCountryCode';
+import { setOrUseParticipations } from '../lib/ab-exclusions';
+
+// Flag to determine whether the canRun function 'setOrUseParticipations' will set a participation (true)
+// or use localstorage participation key to decide canRun result (false)
+const setParticipationsFlag = true;
+
+const EuropeList = [
```
Comment (mxdvl) [inline dotcom-rendering/src/web/experiments/tests/sign-in-gate-mandatory-long-test.ts:9]: Could this be an explicit array of `CountryCode`s?

**Review by mxdvl — APPROVED** [review]
LGTM, though I have not looked at the specific audience numbers.

File: dotcom-rendering/src/web/experiments/tests/sign-in-gate-mandatory-long-test.ts (hunk 1 above)
Comment (vlbee) [reply dotcom-rendering/src/web/experiments/tests/sign-in-gate-mandatory-long-test.ts:9]: Suppose this works `const EuropeList: (CountryCode | '')[] = `

**Review by mxdvl — APPROVED** [review]

**Comment by vlbee** [comment]
@mxdvl Hi. This PR has been updated with new test numbers and the changes introduced by the second test run last wee #6136

It pulls in the changes introduced there: https://github.com/guardian/dotcom-rendering/pull/6136#issuecomment-1270495167

- Fixes in this PR for the new testrun:

-     Change the DCR local storage key from "gu.ab.participations" to "gu.ab._participations"
-     Add an extra ophanRecord firing within the setOrUseParticipations function to capture the moment we bucket a new browser id - &abTestRegister={"abSignInGateMandatoryLongBucketingTestRun - AbBucketed"
-     Switching off the frontend test definition canRun function here

**Pull request merged**